smoketests/missing-packages/webpack.test.js

```diff
@@ -0,0 +1,69 @@
+const path = require('path');
+const execa = require('execa');
+const { renameSync } = require('fs');
+
+const CLI_ENTRY_PATH = path.resolve(__dirname, '../../packages/webpack-cli/bin/cli.js');
+const pathToPackage = require.resolve('webpack').split(path.sep);
```

Comment:
Why we need this?

Reply:
Utility variable which is used later in code for simulating non existence of required package in `node_modules` (`webpack` in this case).

Reply:
> Why just not rename without extra working with paths?
For getting path of `node_modules` folder. But It will always be the root one on CI (as `cwd` is `__dirname`), so I think I can use path.resolve to get to it here. Will try ahead.

Reply:
Tried locally, it works.